src/main/java/by/exadel/internship/team_four/entity/Interview.java

```diff
@@ -0,0 +1,24 @@
+package by.exadel.internship.team_four.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Interview {
+
+    private UUID id;
+    private User admin;
+    private LocalDateTime adminInterviewDate;
+    private String feedback;
```

Comment:
May be, better specify like adminFeedback if you've already specified it for techSpecialist below.

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/112868305-930ca000-90c4-11eb-8479-7890830fa0a9.png)